From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Sat, 22 Feb 2025 05:12:28 +0100
Subject: authList: Expand the list items to use all the available space

So they look as expected and as any other item in the auth prompt


Forwarded: https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/3647
---
 js/gdm/authList.js   | 2 +-
 js/gdm/authPrompt.js | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/js/gdm/authList.js b/js/gdm/authList.js
index 59047df..6575ca4 100644
--- a/js/gdm/authList.js
+++ b/js/gdm/authList.js
@@ -31,7 +31,7 @@ const AuthListItem = GObject.registerClass({
             text,
             style_class: 'login-dialog-auth-list-label',
             y_align: Clutter.ActorAlign.CENTER,
-            x_expand: false,
+            x_expand: true,
         });
 
         super._init({
diff --git a/js/gdm/authPrompt.js b/js/gdm/authPrompt.js
index 0c3eeaf..cf47aa3 100644
--- a/js/gdm/authPrompt.js
+++ b/js/gdm/authPrompt.js
@@ -199,6 +199,8 @@ export const AuthPrompt = GObject.registerClass({
         this._authList = new AuthList.AuthList();
         this._authList.set({
             visible: false,
+            x_align: Clutter.ActorAlign.FILL,
+            x_expand: true,
         });
         this._authList.connect('activate', (list, key) => {
             this._authList.reactive = false;
